extensions/wikia/AdEngine3/src/ad-context.js

```diff
@@ -460,6 +460,10 @@ export default {
 			enabled: false,
 			propertyId: 'd-aIf3ibf0cYxCLB1HTWfBQOFEA',
 		},
+		durationMedia: {
+			enabled: false,
+			siteId: '1066',
```

Comment:
Incorrect ID, Sir. Check epic attachments, please.

Reply:
You right, changed 👍